### PR TITLE
PAYARA-1180 Maven snapshots profile

### PR DESCRIPTION
**Pull request by Cousjava**

### CODE REVIEW

**Comment by Cousjava** [comment]
jenkins test please

File: appserver/distributions/payara-ml/pom.xml [hunk 1]
```diff
@@ -108,6 +116,30 @@
...
+                <property>
+                    <name>DeploySnapshot</name>
+                </property>
+            </activation>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.sonatype.plugins</groupId>
+                        <artifactId>nexus-staging-maven-plugin</artifactId>
+                        <version>1.6.7</version>
```
Comment (arjantijms) [inline appserver/distributions/payara-ml/pom.xml]: Should be 1.6.8 as sonatype always makes a big point of projects using the latest version

**Comment by Cousjava** [comment]
jenkins test please

**Comment by arjantijms** [comment]
jenkins test please

**Comment by Cousjava** [comment]
Jenkins, would you kindly?

**Comment by Cousjava** [comment]
jenkins test please

**Comment by arjantijms** [comment]
Jenkins, would you kindly?

**Pull request merged**